composer.lock

```diff
@@ -6290,16 +6290,16 @@
         },
         {
             "name": "symfony/polyfill-mbstring",
-            "version": "v1.12.0",
+            "version": "v1.13.1",
```

Comment:
@eternoendless If it removed from the composer.json, why is it updated here ?

Reply:
It's required by symfony:

```
$ composer depends "symfony/polyfill-mbstring"
symfony/symfony  v3.4.26  requires  symfony/polyfill-mbstring (~1.0)  
```

Reply:
:ok_hand: